sql/src/main/java/io/crate/action/sql/SQLOperations.java

```diff
@@ -97,6 +97,13 @@ public Session createSession(@Nullable String defaultSchema, @Nullable String us
         return createSession(new SessionContext(defaultLimit, options, defaultSchema, userName));
     }
 
+    public Session createSession(@Nullable String defaultSchema, Set<Option> options, int defaultLimit, String username) {
```

Comment:
why did you create this new overloaded method? you could have just removed the @Nullable annotation from the userName argument of the method above